src/extensions/sections.php

```diff
@@ -32,14 +33,24 @@
                 $value = $this->titleSeparator;
                 return $this->tryResolveQuery($this->titleSeparator);
             },
-            'descriptionFallback' => function () {
-                return $this->tryResolveQuery($this->descriptionFallback);
-            }
+            'defaultTitle' => function () {
+                return $this->tryResolveQuery($this->defaultTitle);
+            },
+            'defaultDescription' => function () {
+                return $this->tryResolveQuery($this->defaultDescription) ?: $this->tryResolveQuery($this->descriptionFallback);
```

Comment:
Maybe simplify the approach. 

```suggestion
                return $this->tryResolveQuery($this->defaultDescription ?: $this->descriptionFallback);
```